pyiceberg/io/__init__.py

```diff
@@ -58,6 +58,8 @@
 S3_PROXY_URI = "s3.proxy-uri"
 S3_CONNECT_TIMEOUT = "s3.connect-timeout"
 S3_SIGNER_URI = "s3.signer.uri"
+S3_SIGNER_ENDPOINT = "s3.signer.endpoint"
+S3_SIGNER_ENDPOINT_DEFAULT_VALUE = "v1/aws/s3/sign"
```

Comment:
nit: convention is `S3_SIGNER_ENDPOINT_DEFAULT`

Reply:
done.